src/test/kotlin/org/rust/ide/annotator/fixes/AddSelfTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.rust.ide.annotator.fixes
+
+import org.junit.Test
+import org.rust.ide.annotator.RsAnnotatorTestBase
+
+class AddSelfTest : RsAnnotatorTestBase() {
+    @Test
+    fun testNoParameters() = checkQuickFix("Add self to function", """
+struct S;
+
+impl S {
+    fn foo() {
+        self/*caret*/;
+    }
+}
+""", """
+struct S;
+
+impl S {
+    fn foo(&self) {
+        self/*caret*/;
+    }
+}
+""")
+
+    @Test
+    fun testHasParameters() = checkQuickFix("Add self to function", """
+struct S;
+
+impl S {
+    fn foo(a: i32) {
+        self/*caret*/;
+    }
+}
+""", """
+struct S;
+
+impl S {
+    fn foo(&self, a: i32) {
+        self/*caret*/;
+    }
+}
+""")
+}
```

Comment:
Could you indent the literals here, using the pattern like in https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/ide/actions/RsJoinRawLinesHandlerTest.kt#L4 ?